src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -92,49 +85,39 @@
-  //   into a sorted map of tally to List of candidate IDs.
-  //   A list is used because multiple candidates may have the same tally.
-  //   This is used to determine when winners are selected and for running tiebreak logic.
+  // Utility function to "invert" the input roundTally map into a sorted map of tally
+  // to List of candidate IDs.  A list is used because multiple candidates may have the same tally.
+  // This is used to determine when winners are selected and for running tiebreak logic.
   // param: roundTally input map of candidate ID to tally for a particular round
   // param candidatesToInclude: list of candidate IDs which may be included in the output.
   //   This filters out candidates when running a tiebreak tabulation which relies
   //   on the tied candidate's previous round totals to break the tie.
   // param: shouldLog is set to log to console and log file
-  // return: sorted map of tally to List of candidate IDs drawn from the input data and excluding
-  //   candidates not appearing in candidatesToInclude)
   static SortedMap<BigDecimal, LinkedList<String>> buildTallyToCandidates(
       Map<String, BigDecimal> roundTally, Set<String> candidatesToInclude, boolean shouldLog) {
-    // output map structure containing the map of vote tally to candidate(s)
     SortedMap<BigDecimal, LinkedList<String>> tallyToCandidates = new TreeMap<>();
     // for each candidate record their vote total into the countToCandidates object
-    // candidate is the current candidate as we iterate all candidates under consideration
     for (String candidate : candidatesToInclude) {
-      // vote count for this candidate
       BigDecimal votes = roundTally.get(candidate);
       if (shouldLog) {
         Logger.log(Level.INFO, "Candidate \"%s\" got %s votes.", candidate, votes.toString());
       }
-      // all candidates in the existing output structure (if any) who received the same vote tally
       LinkedList<String> candidates =
           tallyToCandidates.computeIfAbsent(votes, k -> new LinkedList<>());
-      // new container list for candidates who received this vote tally
       candidates.add(candidate);
     }
     return tallyToCandidates;
   }
 
-  // function: tabulate
-  // purpose: run the main tabulation routine to determine contest results
-  //  this is the high-level control of the tabulation algorithm
+  // run the main tabulation routine to determine contest results
   // returns: set containing winner(s)
   Set<String> tabulate() throws TabulationCancelledException {
     if (config.needsRandomSeed()) {
       Random random = new Random(config.getRandomSeed());
       if (config.getTiebreakMode() == TieBreakMode.GENERATE_PERMUTATION) {
-        // sort candidate permutation for reproducibility
+        // sort candidate permutation first for reproducibility
         Collections.sort(config.getCandidatePermutation());
+        // every day I'm shuffling
```

Comment:
are you, though?

Reply:
you're not always around to see it, but yes